src/components/Dropdown/Dropdown.tsx

```diff
@@ -322,8 +345,14 @@ const Dropdown = ({
 
   shouldStayInView = true,
   intersectionThreshold = 1.0,
+  intersectionContainer = null,
   intersectionObserverPrecision = 100,
   virtualizeOptions = true,
+
+  searchable = true,
```

Comment:
we probably don't want to default to true here - usually filtering is only needed when the user is presented with a long list.

That said if you think the experience is basically the same we can keep it on by default - but what if someone thinks they can type their value into the search box if it's not in the list 🤔 maybe we ought to clear the search string on blur?